ranger/container/settings.py

```diff
@@ -96,6 +96,7 @@
 ALLOWED_VALUES = {
     'cd_tab_case': ['sensitive', 'insensitive', 'smart'],
     'confirm_on_delete': ['multiple', 'always', 'never'],
+    'draw_borders': ['None', 'both', 'outline', 'separators'],
```

Comment:
Since `'None'` here is a string and not the true `None` value, let's keep it lowercase for consistency.